addons/io_hubs_addon/__init__.py

```diff
@@ -0,0 +1,43 @@
+from .io import gltf_exporter
+from . import (nodes, components)
+bl_info = {
+    "name": "Hubs Blender Addon",
+    "author": "Mozilla Hubs",
+    "description": "Tools for developing GLTF assets for Mozilla Hubs",
+    "blender": (2, 92, 0),
+    "version": (0, 1, 0),
```

Comment:
Maybe it's a good time to jump to major?

Reply:
Yeah, I was thinking that though something about being "1.0" feels odd. We haven't really been following semver, but we also didn't really break backwards compat with this. No strong opinion on what it should be, I just know 0.1.0 is not right since we have already used that version.

Reply:
Actually I guess we are breaking compat with architecture kit. Maybe 1.0 would be good.